locales/en-US/experiments.ftl

```diff
@@ -126,6 +142,24 @@ pulseContributors1Title = Firefox UX
 pulseContributors2Title = Firefox Platform Product Intelligence
 pulseContributors3Title = Firefox QA
 pulseEolwarning = The Pulse experiment is ending soon, but the data you provided helped us so much that we want to build the feature in Firefox. We will uninstall this experiment soon.
+sendDescription = Encrypt and send files with a link that automatically expires to ensure your important documents don't stay online forever.
+sendIntroduction = <p>Send lets you upload and encrypt large files (up to 1GB) to share online. When you upload a file, send creates a link to pass along to whoever you want. Each link created by Send will expire after 1 download or 24 hours, and all sent files will be automatically deleted from the Send Server.</p> <p>Unlike other Test Pilot experiments. Send does not require an add-on, and can be used in any modern browser.</p>
+sendMeasurements0 = When you use Send, Mozilla receives an encrypted copy of the file you upload, and basic information about the file, such as filename and file size. Mozilla does not have the ability to access the contents of your encrypted file, and only keeps it for the time or number of downloads indicated.
```

Comment:
contents => content

Here it sounds like you can edit the number of downloads before deletion, is that true? It contradicts all other strings talking about 1 download